ONSdigital/eq-author-prototypes
# File Upload

Adds uploading of JSON file to app.

- survey loads from disk
- also started using [Prettier](https://atom.io/packages/prettier-atom) to enforce code style
- also re-implemented storing state in local storage

How to test

- Click 'load'
- Select schema from data directory in author app
- Check that 'create' and 'design' pages show loaded schema
